ansible/playbooks/registration.yaml

```diff
@@ -97,6 +97,28 @@
         - not_registered_found
         - "(rcg.rc != 0) or (use_suseconnect | bool)"
 
+    - name: Ensure cloud-regionsrv-client is on latest version.
+      community.general.zypper:
+        name: cloud-regionsrv-client
+        state: latest
+      when:
+        - ansible_facts['distribution_major_version'] == "12"
+        - not not_registered_found
```

Comment:
Also this one deserve an explanation:
- The article suggest to update cloud-regionsrv-client for 12sp5 PAYG
- `zypper up` only works if image is registered